src/lang/locale/zh-cn.ts

```diff
@@ -740,44 +740,44 @@ export default {
 
   // These are the string values in the UI for enum values and thus they do not follow the key format as used above
   'enums': {
-    'Title Case': '标题大小写',
+    'Title Case': '每词首字母大写',
     'ALL CAPS': '全部大写',
-    'First letter': '首字母大写',
+    'First letter': '仅首字母大写',
     '.': '.', // leave as is
     ')': ')', // leave as is
-    'ERROR': '错误',
-    'TRACE': '跟踪',
-    'DEBUG': 'debug',
-    'INFO': '信息',
-    'WARN': '警告',
-    'SILENT': '静默',
+    'ERROR': 'ERROR',
+    'TRACE': 'TRACE',
+    'DEBUG': 'DEBUG',
+    'INFO': 'INFO',
+    'WARN': 'WARN',
+    'SILENT': 'SILENT',
```

Comment:
Why did these change to their English values instead of being their Chinese equivalents?

Reply:
Just no need, programmers prefer to use the original English of programming terms, and I think debug level is only for programmers.

Reply:
These options are not just meant for developers as it is something that is supposed to be used when reporting issues as well.

Reply:
If that does not change anything, I am fine with merging this.

Reply:
If there is any explanation of different debug level, I think translate these to Chinese will be meaningful, but we just get some words here, so keep it not translated is fine.

Reply:
Gocha. I will go ahead and merge this then.